content/coop.html

```diff
@@ -253,7 +253,7 @@ <h3> Favourite Co-op Experiences </h3>
   <div class="vb"></div>
   <p>"Too many to pick. A funny one is once someone leaked [company X's] internal uber/lyft codes and we were taking unlimited $500 Uber Luxe XLs across California."</p>
   <div class="vb"></div>
-  <p>"Most of them were great but my time in California was definitely the most memorable since I met so many new people and experienced a lot of fun things for the first time. Great weather, great activities, great trips to new cities and great people made for an exciting coop term!"</p>
+  <p>"Most of them were great but my time in California was definitely the most memorable since I met so many new people and experienced a lot of fun things for the first time. Great weather, great activities, great trips to new cities and great people made for an exciting co-op term!"</p>
```

Comment:
```suggestion
  <p>"Most of them were great but my time in California was definitely the most memorable since I met so many new people and experienced a lot of fun things for the first time. Great weather, great activities, great trips to new cities and great people made for an exciting [co-op] term!"</p>
```